x/auth/ante/sigverify.go

```diff
@@ -45,6 +57,14 @@ func init() {
 // This is where apps can define their own PubKey
 type SignatureVerificationGasConsumer = func(meter storetypes.GasMeter, sig signing.SignatureV2, params types.Params) error
 
+// TODO HV2: MainTxMsg has no usage so far. Double check it by looking at heimdall's code.
```

Comment:
I looked up each function of this interface in old Heimdall, and found nothing. Seems to me interface `MainTxMsg` is unused in old Heimdall, so could be deleted (to confirm, you can try removing it from old Heimdall, and build and running UTs to see if there's a build error somewhere)

Reply:
Resolved